src/commands/createRepo/RepoNameStep.ts

```diff
@@ -15,7 +16,8 @@ export class RepoNameStep extends AzureWizardPromptStep<IStaticWebAppWizardConte
     public async prompt(wizardContext: IStaticWebAppWizardContext): Promise<void> {
         wizardContext.newRepoName = (await ext.ui.showInputBox({
             prompt: localize('AppServicePlanPrompt', 'Enter the name of the new GitHub repository. Special characters will be replaced with "-" upon creation.'),
-            validateInput: async (value: string): Promise<string | undefined> => await this.validateRepoName(wizardContext, value)
+            validateInput: async (value: string): Promise<string | undefined> => await this.validateRepoName(wizardContext, value),
+            value: workspace.workspaceFolders ? workspace.workspaceFolders[0].name : undefined
```

Comment:
It bugs me when the default value used is immediately invalid, which is possible here. Can we either:
1. Convert invalid names to valid ones (shorten it, replace special characters, etc.)
2. Only use a default value if we detect it _is_ valid

Reply:
I went with option 2. Part of the validation is that the repo name is available and it seemed simpler to just not show a default value than try to generate a unique repo name